ethcore/types/src/receipt.rs

```diff
@@ -157,6 +157,10 @@ pub struct LocalizedReceipt {
 	pub log_bloom: Bloom,
 	/// Transaction outcome.
 	pub outcome: TransactionOutcome,
+	/// Receiver address
+	pub to: Option<H160>,
```

Comment:
Note we now have:
```
contract_address: Option<_>,
to: Option<_>,
```
and exactly one of the two will be `Some` for every transaction.

Just making sure that what's speced.